<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>MainWindow</class>
  <widget class="QMainWindow" name="MainWindow">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>1200</width>
        <height>800</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Simulation</string>
    </property>
    <widget class="QWidget" name="centralwidget">
      <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
          <widget class="PlotWidget" name="vt_graph" native="true">
            <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
          </widget>
        </item>
        <item row="1" column="1">
          <widget class="PlotWidget" name="av_graph" native="true">
            <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
          </widget>
        </item>
        <item row="3" column="0">
          <widget class="PlotWidget" name="at_graph" native="true">
            <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
          </widget>
        </item>
        <item row="3" column="1">
          <widget class="PlotWidget" name="cd_graph" native="true">
            <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
          </widget>
        </item>
      </layout>
    </widget>
  </widget>

  <customwidgets>
    <customwidget>
      <class>PlotWidget</class>
      <extends>QWidget</extends>
      <header>pyqtgraph</header>
      <container>1</container>
    </customwidget>
  </customwidgets>
  <resources/>
  <connections/>
</ui>
